### DISCUSSION TITLE
Vs Mario port by me

### DISCUSSION THREAD

**Discussion by StrYk85** [luckydog7/Funkin-android#1620] [mods]
Wow the second port this month
Here:https://drive.google.com/file/d/1M5lpx1fQJbFTKz_TSN0rUcEHN_BAxtsT/view?usp=drivesdk

Here are some screenshots:
![Screenshot_20210707-135650](https://user-images.githubusercontent.com/79270509/124770105-b99f6a00-df42-11eb-8d9f-cb91e717a766.jpg)
![Screenshot_20210707-135720](https://user-images.githubusercontent.com/79270509/124770120-bc9a5a80-df42-11eb-9249-c489545ae014.jpg)
![Screenshot_20210707-135731](https://user-images.githubusercontent.com/79270509/124770141-c15f0e80-df42-11eb-9b09-46be0e6582f4.jpg)
![Screenshot_20210707-135750](https://user-images.githubusercontent.com/79270509/124770159-c58b2c00-df42-11eb-9669-9e025b02aa44.jpg)

The custom week template is by Iprimegamerz
His channel:https://m.youtube.com/channel/UCpWyByigLOl3Cnzux157CHw
My channel:https://www.youtube.com/channel/UCH3Cg6ks7yErGWtMiISiOog

**Comment by Sonic467** [comment]
This is great! Thanks!
